update_bench_traces_rel_yaml.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+
+import gobexec.main
+from gobexec.goblint.bench import txtindex
+from gobexec.goblint.result import AssertTypeSummary, YamlSummary
+from gobexec.goblint.tool import GoblintTool
+
+from gobexec.model.result import TimeResult
+from gobexec.model.tools import ExtractTool
+from gobexec.output.renderer import FileRenderer, ConsoleRenderer, MultiRenderer
+
+goblint_witness = GoblintTool(
+    name="witness_gen",
+    program=str(Path("../analyzer/goblint").absolute()),
+    args=["--conf", str(Path("../analyzer/conf/traces-rel.json").absolute()), "--enable", "dbg.debug", "--enable",
+          "witness.yaml.enabled"],

```

Comment:
Adding these arguments for witness generation should generate some non-empty witnesses:
```console
--set ana.activated[+] apron --set ana.path_sens[+] threadflag --set ana.relation.privatization mutex-meet-tid-cluster12 --set witness.yaml.entry-types[-] invariant_set
```